src/types/configuration-types.ts

```diff
@@ -101,13 +95,7 @@ export const BotConfigSchema = StrictObject(
       basePriceMultiplier: T.Number({ default: 1 }),
       issueCreatorMultiplier: T.Number({ default: 1 }),
     }),
-    commands: T.Array(
-      StrictObject({
-        name: T.String(),
-        enabled: T.Boolean(),
-      }),
-      { default: allCommands }
-    ),
+    disabledCommands: T.Array(T.String(), { default: allCommands }),
```

Comment:
Perhaps it makes sense to enable all commands by default? It seems that disabling by default has always caused more headaches when starting new repos for us? rfc @whilefoo